### ISSUE TITLE
NTR: Histamine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8848]
New term request Histamine high in urine (elthat_111113035301):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000532
- label: Elevated urinary histamine level
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:890980
- def: The amount of histamine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: Urinary histamine excretion may be abnormally high in a variety of clinical settings including systemic mastocytosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8848
- synonyms:
